### ISSUE TITLE
Rule Change: no-warning-comments support multiline comments/continuations

### ISSUE THREAD

**Issue by brettwillis** [eslint/eslint#20197]
### What rule do you want to change?

no-warning-comments

### What change do you want to make?

Generate fewer warnings

### How do you think the change should be implemented?

A new default behavior

### Example code

```js
// TODO: Some really long explanation
//  that continues on another line with an indent
```

### What does the rule currently do for this code?

It only highlights the first line, not the second line, which creates a visual implication that the two lines are logically separate.

### What will the rule do after it's changed?

Lines which follow the initial line with indentation should be grouped and highlighted as the same problem/issue.

Because the second line is not highlighted it creates a visual implication that the two lines are logically separate. So I find myself doing this, below, but the downside is that it now creates multiple lint "problems".

```js
// TODO: Some really long explanation
// TODO: that continues on another line with an indent
```

### Participation

- [ ] I am willing to submit a pull request to implement this change.

### Additional comments

Using indentation like this to imply continuation is a convention that other IDEs/extensions follow. IntelliJ IDEA is one such example.

**Comment by mdjermanovic** [comment]
Hi @brettwillis! Thanks for the suggestion, but as stated in the [docs](https://eslint.org/docs/latest/rules/no-warning-comments) this rule is frozen and is not accepting feature requests. You can always copy a core rule into a custom rule and modify it.
